Dockerfile

```diff
@@ -11,7 +11,12 @@ RUN apt-get update && \
     pip install opencv-python
 
 RUN pip uninstall -y pycocotools && \
-    pip install pybind11
+    pip install pybind11 && \
+    pip install scikit-image
+    
+RUN wget https://github.com/aws-samples/mask-rcnn-tensorflow/releases/download/v0.0.0/example_log.tar.gz && \
+    tar -xzf example_log.tar.gz example_log
```

Comment:
What is this for? Are you sure it can run? When I ran it in the container I encountered:
```
tar: example_log: Not found in archive
```

Reply:
This downloads a pretrained model for visualization. Will do some testing to see if it needs to be updated.

Reply:
Problem was an error in the tar file itself. Uploaded a new one. Can you try again?